pennylane/transforms/tape_expand.py

```diff
@@ -518,3 +560,19 @@ def circuit():
 
             finally:
                 dev.preprocess = original_preprocess
+    else:
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                action="ignore",
+                message=r"max_expansion argument is deprecated",
+                category=qml.PennyLaneDeprecationWarning,
```

Comment:
Can we use the `exceptions` module 😄 
```suggestion
                category=qml.exceptions.PennyLaneDeprecationWarning,
```

Reply:
realized we could get rid of these entirely now.